addon/utils/is-valid-phone-number.js

```diff
@@ -0,0 +1,11 @@
+import CountryCodes from 'ember-cli-uniq/enums/country-codes-type';
+import PhoneNumberValidations from 'ember-cli-uniq/enums/phone-number-validation-type';
+
+export default function isValidPhoneNumber(countryCode, number) {
+  switch (countryCode) {
+    case CountryCodes.PT:
```

Comment:
I think it would be easier to add validations in future if you do something like:

```
return PhoneNumberValidations[countryCode] 
  ? PhoneNumberValidations[countryCode].test(number)
  : true
```
And remove the switch
This way if you want to add a validation for a new country you just have to add the `regEx`on the enum (not needing to change the switch to add a new case). 
It will be one less file to change if you want to add a new validation, just because of that :)